src/main/exception.c

```diff
@@ -244,6 +244,15 @@ PyObject *AerospikeException_New(void)
     PyObject_SetAttrString(exceptions_array.NoResponse, "code", py_code);
     Py_DECREF(py_code);
 
+    // max errors limit reached, AEROSPIKE_MAX_ERROR_RATE, -14
+    exceptions_array.MaxErrorRateExceeded = PyErr_NewException(
```

Comment:
The documentation needs to include this exception: https://aerospike-python-client.readthedocs.io/en/latest/exception.html#client-errors

Reply:
Added documentation at 660c5a9deb0bc3d912991508ee80bb6b16f523a8